components/Player.js

```diff
@@ -0,0 +1,60 @@
+import propTypes from 'prop-types'
+import Card from './Card'
+
+/**
+ * Player list component
+ */
+export default function Player(props) {
+  return (
+    <div className="flex md:flex-col rounded border border-slate-300 my-2">
+      <div className=" flex w-full justify-between">
+        <div className="flex ">
+          <div
+            className={`flex items-center justify-center rounded-lg p-2  ${props.bgColor}`}
```

Comment:
`rounded-lg` and `props.bgColor` can be removed since I don't think we will use them.
Passed red as prop. (We can use the `props.bgColor` below instead of hardcoding `bg-slate-300` )
![image](https://user-images.githubusercontent.com/77116011/157708251-6af81ebd-29ac-4c3a-a1ee-702da6c9062f.png)

Reply:
I am working on updating that bit. it was more of a placeholder.